src/Lecture1.hs

```diff
@@ -119,4 +133,17 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list =
+  let lower = show (count n list (<))
+      greater = show (count n list (>))
+   in show n ++ " is greater than " ++ greater ++ " elements and lower than " ++ lower ++ " elements"
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
You can write it more performant with only one traversal of the list 🙂
It's desirable to write functions that traverse lists single time and consume elements only once. In that case, the composition of such functions will have nice streaming and lazy properties.

Reply:
Not only haskell is lazy, I am too :) I wrote a shorter solution and let the computer to do the extra work 😄  
Refactored, thanks for the review 👍